### PR TITLE
Adding the Obscuro testnet

### PR DESCRIPTION
**Pull request by CaisManai**
Adding the Obscuro testnet running as an L2 on Sepolia

### CODE REVIEW

File: _data/chains/eip155-443.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "symbol": "ETH",
+    "decimals": 18
+  },
+  "rpc": ["https://testnet.obscu.ro/"],
+  "faucets": [],
+  "infoURL": "https://obscu.ro/",
+  "explorers": [
+    {
+      "name": "Obscuro Sepolia Rollup Explorer",
+      "url": "https://testnet.obscuroscan.io/",
```
Comment (ligi) [inline _data/chains/eip155-443.json:19]:
```suggestion
      "url": "https://testnet.obscuroscan.io",
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Review by ligi — CHANGES_REQUESTED** [review]
https://testnet.obscuroscan.io is down

**Comment by CaisManai** [comment]
> https://testnet.obscuroscan.io is down

apologies @ligi 🙏 it's up now

File: _data/chains/eip155-443.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "decimals": 18
+  },
+  "rpc": ["https://testnet.obscu.ro"],
+  "faucets": [],
+  "infoURL": "https://obscu.ro",
+  "explorers": [
+    {
+      "name": "Obscuro Sepolia Rollup Explorer",
+      "url": "https://testnet.obscuroscan.io",
+      "standard": "EIP3091"
```
Comment (ligi) [inline _data/chains/eip155-443.json:20]:
```suggestion
      "standard": "none"
```

**Review by ligi — CHANGES_REQUESTED** [review]
explorer seems EIP3091 incompatible

**Comment by CaisManai** [comment]
> explorer seems EIP3091 incompatible

Apologies for this @ligi. Also, our bridge is still not fully up but isn't required for testnet. Should I remove this from the PR or can I keep it in for now?

**Pull request merged**